src/Frame.tsx

```diff
@@ -1,6 +1,6 @@
 import clsx from "clsx";
 
-let paddingMap = { none: "", md: "p-8" };
+const paddingMap = { none: "", md: "p-8" };
```

Comment:
@mintlify/engineering as a general rule, use const unless you need to mutate it.

> As a general rule, you should always declare variables with const, if you realize that the value of the variable needs to change, go back and change it to let.

https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa

Reply:
Looks like it was a copy-paste error from the MVP days haha

Reply:
dw, this will be an automatic linter fix when we get linting in this repo 😁